doc/sphinx/how-to/index.md

```diff
@@ -1,12 +1,15 @@
 # How-to guides
-These how-to guides cover the key aspects of working with Miaspects of working with Mir.
+These how-to guides cover the key aspects of working with Mir.
 
 - [Developing a Compositor](developing-a-wayland-compositor-using-mir): build your own, custom compositor with Mir
 - [Developing protocol extensions for Mir](developing-wayland-extension-protocols-for-mir-servers): extend your compositor with support for a custom protocol extension
 - [Getting involved in Mir](getting_involved_in_mir): first steps to contributing to Mir
 - [Using checkbox-mir](how-to-use-checkbox-mir): validating your environment using [checkbox-mir](https://snapcraft.io/checkbox-mir)
 - [Calibrating a touchscreen device](how-to-calibrate-a-touchscreen-device): creating and using a calibration matrix using [the libinput snap](https://snapcraft.io/libinput)
 - [Enabling graphics-core22 on a device](how-to-enable-graphics-core22-on-a-device): creating a `graphics-core22` provider snap for a new board
+- [How to specify start up applications](..//how-to/how-to-specify-startup-apps.md)
+- [How to handle user input](../how-to/how-to-handle-keyboard-input.md)
+- [How to Control Compositor Responses to Client Decoration Requests](../how-to/how-to-control-compositor-responses-to-client-decoration-requests.md)
```

Comment:
This title (also used elsewhere and for slug) is hard to parse.

Perhaps: "Negotiating client/server side decorations" (not great I admit)

Reply:
I also prefer (as you can see above) "Specifying…", "Handling…" in the link titles, otherwise we'll have "How to" most of the page content :)

You also don't need `../how-to/`.

"Deciding between client / server side decorations"…?
"Preferring client or server side decorations"…?
"Selecting client / server decoration preference"…?

Reply:
Or... can we assume some domain knowledge in anyone looking for this?

"Setting CSD/SSD preference"

Reply:
I took a bit of both suggestions :) 

A small note about the new file name. I couldn't bring myself to prefix it with `how-to-` since it read too awkwardly. I don't think it's a big issue other than breaking the uniformity of naming in the how-to directory.